welcome/learn_more_about_openshift.adoc

```diff
@@ -30,9 +30,6 @@ Use the following sections to find content to help you learn about and use {prod
 |
 | xref:../backup_and_restore/index.adoc#backup-restore-overview[Backup and restore]
 |
-
-|
-| xref:../scalability_and_performance/recommended-install-practices.adoc#recommended-cluster-install-practices[Performance and scalability]
```

Comment:
I'm having trouble finding this table link in any of the currently built versions of the doc, so I can't compare the tables before and after. Could you link me? 🙂

Reply:
@jldohmann So strange! I can't seem to find it either. :thinking: It's not in any of the published docs.